tests/sentry/lang/java/test_plugin.py

```diff
@@ -18,6 +18,30 @@
     69:69:java.lang.Class[] getExtraClassContext() -> a
     65:65:void <init>(org.slf4j.helpers.Util$1) -> <init>
 """
+PROGUARD_INLINE_UUID = "d748e578-b3d1-5be5-b0e5-a42e8c9bf8e0"
+PROGUARD_INLINE_SOURCE = b"""\
+# compiler: R8
```

Comment:
I'd add a test using Proguard as well, R8 is a replacer for Proguard, but still many people didn't upgrade yet and they have slight differences, maybe generate a mapping file using Proguard and Kotlin instead of Java

Reply:
Ideally, I would like to add those to https://github.com/getsentry/rust-proguard directly. Maybe you can guide me through how to set that stuff up ;-)

Reply:
yep wfm.